site/documentation/content/api/Metrics.md

```diff
@@ -75,6 +75,7 @@ Metrics provided by the protocol adapters are:
 | *hono.connections.authenticated*   | Gauge               | *host*, *component-type*, *component-name*, *tenant*                                         | Current number of connected, authenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.unauthenticated* | Gauge               | *host*, *component-type*, *component-name*                                                   | Current number of connected, unauthenticated devices. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
 | *hono.connections.authenticated.duration* | Timer        | *host*, *component-type*, *component-name*, *tenant*                                         | The overall amount of time that authenticated devices have been connected to protocol adapters. <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
+| *hono.connections.attempts.total*  | Counter             | *host*, *component-type*, *component-name*, *tenant*                                         | The number of connection attempts to protocol adapters. Can be distinguished by outcome by evaluating the tag "outcome" (and its corresponding "reason"). <br/> **NB** This metric is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric. |
```

Comment:
FMPOV it should also have a tag *status* or *outcome* which indicated whether the attempt succeeded or, if not, why. Possible values of the tag could therefore be *succeeded*, *rejected* and *unauthorized* FMPOV ...

Reply:
I'm confused :) Possibly my phrasing is misleading, but it states: "Can be distinguished by outcome by evaluating the tag "outcome" (and its corresponding "reason")".

So there's one tag ("outcome") indicating the result (succeeded, rejected, etc.) and another one ("reason") which can be used for detailing the reason for the outcome (e.g. by the value "adapter-resource-limit-exceeded").

Reply:
Each tag adds another dimension when evaluating meters in the backend. If there is no compelling reason to use two separate tags for outcome and reason I would suggest to stick with a single tag instead. For a successful outcome (connection established) there will never be a reason anyway, so filtering for successful/unsuccessful outcomes should be easy enough, right?

Reply:
Good point. Since the Prometheus doc themselves warns about it

>CAUTION: Remember that every unique combination of key-value label pairs represents a new time series, which can dramatically increase the amount of data stored. Do not use labels to store dimensions with high cardinality (many different label values), such as user IDs, email addresses, or other unbounded sets of values.

(https://prometheus.io/docs/practices/naming/)

I guess it's not worth having two labels for clarity. Changed with 25ce759.